stream-chat-android-ui-common/src/main/kotlin/com/getstream/sdk/chat/adapter/MessageListItem.kt

```diff
@@ -24,16 +22,13 @@ import java.util.zip.Checksum
 public sealed class MessageListItem {
 
     public fun getStableId(): Long {
-        val checksum: Checksum = CRC32()
-        val plaintext = when (this) {
-            is TypingItem -> "id_typing"
-            is ThreadSeparatorItem -> "id_thread_separator"
-            is MessageItem -> message.id
-            is DateSeparatorItem -> date.toString()
-            is LoadingMoreIndicatorItem -> "id_loading_more_indicator"
+        return when (this) {
+            is TypingItem -> TYPING_ITEM_STABLE_ID
+            is ThreadSeparatorItem -> THREAD_SEPARATOR_ITEM_STABLE_ID
+            is MessageItem -> message.id.hashCode().toLong()
```

Comment:
The only possible problem is a collision in these hashcodes, but I really hope that that won't be an issue in practice.

Reply:
Yes, hope so, otherwise we'll have to add additional filed to Message class which will be calculated only once.